infra/terraform/environments/dev/main.tf

```diff
@@ -234,4 +234,13 @@ module "service" {
       listener_rule_host_header = "ssweb.*"
     }
   }
+
+  job_definitions = {
+    "processQueue" = {
+      job_name = "processQueue",
```

Comment:
Would it be a good idea to use the `each.key` and remove the `job_name`? It will remove a lot of future duplication when we have dozens of jobs.